### ISSUE TITLE
Extension insists on credentials

### ISSUE THREAD

**Issue by ugurhicyilmam** [bogaziciswe/b.w.a.t#110]
When extension is loaded, it consistently insists on credentials. I believe, a basic authentication window should not appear at all.

**Comment by ugurhicyilmam** [comment]
Since the server responds with 401 in case of unauthorized access, browser shows a basic authentication window. A quick solution would be changing server's default unauthorized 401 response to 400 or even 200. Would it be ok for client application @Mephala, @oakbas, @sarahbe, @ozkuran?

**Comment by uskudarli** [comment]
Are the credentials required -- registered accounts for your plugin or something else?

**Comment by uskudarli** [comment]
I tried to register again, but have the same problem.

I am not able to test your project, unless I can register and test annotations. What do you suggest?

**Comment by ugurhicyilmam** [comment]
@uskudarli According to logs, you are trying to register with a password shorter than 6 characters. There is a bug about this on extension, which tells you to choose a password between 3 and 30 characters. I am creating an issue for this.

edit: Password length issue: #111 

edit 2: You can try to register with your email address and with a password at least 6 characters, sorry about this silly design bug.

**Comment by uskudarli** [comment]
there is no instructions about restrictions that I can see -- is there?

**Comment by uskudarli** [comment]
I tried logging in with a longer password. I still get same error? I can't see any feedback about why it fails.

**Comment by ugurhicyilmam** [comment]
A valid password should be between 6 and 16 characters.

Yes, there are instructions in form as placeholders on inputs of registration form. However, the instruction about password wasn't compatible with our requirements anyway. Mahmut has just fixed it. There is an issue about it: #111

**Comment by uskudarli** [comment]
i will update and try again.

**Comment by uskudarli** [comment]
I updated reinstalled and tried again. Still same error message.

I think you should give some error as response as to what is going on.

[edited to add] : tried on both firefox and chrome

**Comment by uskudarli** [comment]
are you able to register with new accounts? have you validated it?

**Comment by uskudarli** [comment]
I also tried with the user credentials within your test document for ozkuran. Still get the message:

ErrorCode -> BWAT014: Bad credentials

**Comment by ugurhicyilmam** [comment]
@uskudarli Yes, I have registered multiple times today without an issue. I can register right now. According to logs, your registration succeed using "uskudarli@gmail.com", perhaps you may try to login again.

**Comment by uskudarli** [comment]
So far, I only have seen the popup error. 

I will try to see if by chance I am logged in now.

How should I know I successfully registered, if I get a popup error? Do you never see that message?
Is the related to the aws credential request?

**Comment by uskudarli** [comment]
Although I get error messages, I was able to get the popup for textual annotations.

When I submit, I a popup asking for aws credentials. Which I have asked you about. If this is different from the plugin user registration, I do not have any other credentials. I maybe missed it in your documentation. Can you point me to the page number so I can find it?

**Comment by uskudarli** [comment]
got your mail now. will test again.

But, still, please lead me to the instructions in your documentation.

**Comment by ugurhicyilmam** [comment]
@oakbas has just solved the popup issue. It asks for your own credentials but it should not have been appeared in the first place. It is not a feature of our application.

When registration succeed, user automatically logs in. A success message would be better. I will open an issue to add a success message on registration success.

**Comment by uskudarli** [comment]
I just tried and have some luck with logging in finally.

**Comment by uskudarli** [comment]
OK, got my first annotation, and it validates. So, onwards...

Do you not include the "creator" of the annotation? I was expecting to see it as my annotation.

**Comment by ugurhicyilmam** [comment]
Yes we include it when persisting the annotation. You can see the list of annotations by clicking the icon of our extension. You may refer to page 14. of [user manual. ](https://github.com/bogaziciswe/b.w.a.t/blob/master/reportsAndLogs/Reports/UserManual/UserManual.docx)
